colearn_grpc/example_mli_factory.py

```diff
@@ -23,14 +23,6 @@
 from colearn_grpc.mli_factory_interface import MliFactory
 from colearn_grpc.factory_registry import FactoryRegistry
 
-# These are imported to they are registered in the FactoryRegistry and are available here
-# pylint: disable=W0611
-import colearn_keras.keras_mnist  # type:ignore # noqa: F401
-import colearn_keras.keras_cifar10  # type:ignore # noqa: F401
-import colearn_pytorch.pytorch_xray  # type:ignore # noqa: F401
-import colearn_pytorch.pytorch_covid_xray  # type:ignore # noqa: F401
-import colearn_other.fraud_dataset  # type:ignore # noqa: F401
-
```

Comment:
Moved these to where the server is created, so that you can change which tasks are registered.